build.sbt

```diff
@@ -38,11 +37,11 @@ lazy val core = project
       "eu.timepit"                 %% "refined-scalacheck"         % refinedVersion,
       "org.typelevel"              %% "cats-effect"                % catsEffectVersion,
       "org.mongodb.scala"          %% "mongo-scala-driver"         % mongoVersion,
-      "com.slamdata"               %% "qdata-json"                 % qdataVersion,
+      "com.slamdata"               %% "quasar-foundation"          % quasarVersion,
+
```

Comment:
what's the reason for this dep change? and how come `qdata-json` `Test` isn't required anymore?

Reply:
Removing transient dependency -- lesser chance to have incompatible dependencies.
As far as I understand ds-mongo doesn't use anything from qdata-Test.

Reply:
I usually add a lib as dependency to the build as soon as that lib is used directly.

But OTOH this saves an `autobump` PR, so I think I'm ok with this. 

BTW @cryogenian has other (opposite) strategy (quote from discord):
> I personally prefer to have as few dependencies as I can

Opinions on this, @djspiewak or anyone else?

Reply:
@rintcius I think we've been mostly settling on leaning on transitive dependencies whenever we can, since they're there anyway. Obviously `Test` dependencies can't be done transitively in this way.

In this case though, eyeballing the diff above, it looks like a `qdata` dependency was converted into a `quasar` dependency. Does this project actually use anything from `quasar-foundation`? If not, then the dependency should still be on `qdata-json`. However, if `quasar-foundation` is required for other reasons, then it's fine to pick up `qdata-json` through it.

Reply:
Yes, It's `Disposable`

Reply:
Ok, if we are doing transitive deps then I think we can get rid of `quasar-foundation` too I guess? (because that's already pulled in because via `quasar-connector`)

Reply:
Removed, `quasar-foundation-Test` is needed because of `>>*` and friends.

Reply:
In that case, then I think the diff as-written is correct, IMO, since you already need `quasar-foundation` in all these ways, so there's no need to redundantly depend upon `qdata-json`.